scripts/bundleSize/getBundleData.js

```diff
@@ -136,6 +136,9 @@ const getServiceThemeBundleData = () =>
       ),
     }));
 
-exports.getPageBundleData = getPageBundleData;
-exports.getServiceConfigBundleData = getServiceConfigBundleData;
-exports.getServiceThemeBundleData = getServiceThemeBundleData;
+const _getPageBundleData = getPageBundleData;
+export { _getPageBundleData as getPageBundleData };
+const _getServiceConfigBundleData = getServiceConfigBundleData;
+export { _getServiceConfigBundleData as getServiceConfigBundleData };
+const _getServiceThemeBundleData = getServiceThemeBundleData;
+export { _getServiceThemeBundleData as getServiceThemeBundleData };
```

Comment:
Let's remove this, and use export const for all of the functions we want to export above

```suggestion
```